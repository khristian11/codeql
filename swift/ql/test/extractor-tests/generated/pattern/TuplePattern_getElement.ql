import codeql.swift.elements
import TestUtils

from TuplePattern x, int index
where toBeTested(x)
select x, index, x.getElement(index)
