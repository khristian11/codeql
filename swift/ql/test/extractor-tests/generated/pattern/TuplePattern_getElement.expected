| patterns.swift:4:9:4:17 | (...) | 0 | patterns.swift:4:10:4:10 | x |
| patterns.swift:4:9:4:17 | (...) | 1 | patterns.swift:4:13:4:13 | y |
| patterns.swift:4:9:4:17 | (...) | 2 | patterns.swift:4:16:4:16 | z |
| patterns.swift:12:14:12:21 | (...) | 0 | patterns.swift:12:15:12:15 | xx |
| patterns.swift:12:14:12:21 | (...) | 1 | patterns.swift:12:19:12:19 | yy |
| patterns.swift:28:18:28:23 | (...) | 0 | patterns.swift:28:19:28:19 | i |
| patterns.swift:28:18:28:23 | (...) | 1 | patterns.swift:28:22:28:22 | s |
